firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/FirebaseAppDistributionLifecycleNotifier.java

```diff
@@ -25,12 +29,16 @@
 import com.google.android.gms.tasks.Task;
 import com.google.android.gms.tasks.TaskCompletionSource;
 import com.google.android.gms.tasks.Tasks;
+import com.google.firebase.annotations.concurrent.UiThread;
 import com.google.firebase.appdistribution.FirebaseAppDistributionException;
 import java.util.ArrayDeque;
 import java.util.Queue;
 import java.util.concurrent.Executor;
+import javax.inject.Inject;
+import javax.inject.Singleton;
 
-class FirebaseAppDistributionLifecycleNotifier implements Application.ActivityLifecycleCallbacks {
+@Singleton // Only one lifecycle notifier is required across the entire app
+class FirebaseAppDistributionLifecycleNotifier {
```

Comment:
Please review. This was updated to support an "ignored" class when getting the current activity. This is so we can take a screenshot of the previous activity while `TakeScreenshotAndStartFeedbackActivity` is active.